source/stylesheets/variables.scss

```diff
@@ -1,9 +1,9 @@
-
 // typography
 $base-line-height: 1.4;
 $base-font-size: 1.2em;
 
-
 // colors
 $app-background-color: #fff;
 $app-font-color: #000;
+
+$app-color: #333;
```

Comment:
This variable isn't very descriptive. Can you pick a name that more clearly communicates what this color is used for?